_posts/2024-12-20-building-databases-for-the-right-user.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: post
+title: Building databases for the right user
```

Comment:
This is a pretty tame title. Initially I made it "Six decades of building databases for the wrong user" which feels too dramatic and more complainy than solutiony